From: =?utf-8?b?Ik1hcmNvIFRyZXZpc2FuIChUcmV2acOxbyki?= <mail@3v1n0.net>
Date: Wed, 26 Feb 2020 06:43:54 +0100
Subject: meson: Depend on gjs 1.58.1

In ubuntu we provide patches to work with such version as well
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 910a3e9..5ecbc36 100644
--- a/meson.build
+++ b/meson.build
@@ -23,7 +23,7 @@ eds_req = '>= 3.17.2'
 gcr_req = '>= 3.7.5'
 gio_req = '>= 2.56.0'
 gi_req = '>= 1.49.1'
-gjs_req = '>= 1.63.2'
+gjs_req = '>= 1.58.1'
 gtk_req = '>= 3.15.0'
 mutter_req = '>= 3.35.91'
 polkit_req = '>= 0.100'
